grpc/src/main/java/com/linecorp/armeria/server/grpc/UnaryServerCall.java

```diff
@@ -95,26 +94,12 @@
 
     @Override
     public void request(int numMessages) {
-        if (ctx.eventLoop().inEventLoop()) {
-            request();
-        } else {
-            ctx.eventLoop().execute(this::request);
-        }
-    }
-
-    private void request() {
-        if (listener() == null) {
-            return;
-        }
-        startDeframing();
+        // The request of unary call is not reactive. `startDeframing()` will push the request message to
+        // the stub through `listener.onMessage()` after `listener.onReady()` is invoked.
```

Comment:
On second thought, we can safely ignore `request(int)` for unary calls.
The input message of a unary call is not wrapped with `StreamObserver` so we can pass the decoded object to the listener for simplicity.